Assets/Mirror/Runtime/NetworkConnectionToServer.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Mirror
+{
+    public class NetworkConnectionToServer : NetworkConnection
+    {
+        public NetworkConnectionToServer(string networkAddress) : base(networkAddress)
```

Comment:
why does client's connection need networkAddress?

Reply:
I asked you the same thing in discord.   It is not needed by the client or server

Reply:
it's definitely not needed by client. 
I can see it being useful on server for GMs.
but might as well get it from Transport where needed.
feel free to remove it. makes other code easier too.

Reply:
Done